documentation/release-notes/topics/version_5.md

```diff
@@ -0,0 +1,75 @@
+# Version 5
+
+## Hyperview 5.0 (TBD)
+
+This section covers significant changes and bug fixes in Hyperview 5.0.x since version 4.9.x
+
+:::{important}
+1. This release has changes and improvements to the Data Collector software. We strongly recommend updating
+   your installed Data Collectors to the latest version to maintain an optimal monitoring and discovery
+   experience.
+
+2. Customers using Hyperview open-source tools such as asset tool must update to the latest version.
+:::
+
+### New Feature: AI Assistant Beta
+
+An **opt-in** AI assistant beta is being introduced in this version of Hyperview. Hyperview Administrators are able to enable this functionality. Once enabled, Hyperview users can use it to get information about the application, navigate, search and perform read only actions.
```

Comment:
Sometimes we write, "AI Assistant" and sometimes we write "AI assistant". We should be consistent.

Reply:
Done, Thanks!